system/jlib/jsocket.cpp

```diff
@@ -3618,7 +3618,7 @@ class CSocketBaseThread: public Thread
     bool terminating;
     CriticalSection sect;
     Semaphore ticksem;
-    atomic_t tickwait;
+    std::atomic_uint tickwait;
```

Comment:
Could possibly.  I erred on the side of safety for the changes that I thought were unlikely to be time critical.